src/api/spec/support/shared_examples/features/bootstrap_user_tab.rb

```diff
@@ -52,7 +52,12 @@ def toggle_checkbox(html_id)
     end
 
     scenario 'Add non existent user' do
-      click_link('Add User')
+      # TODO: Remove once responsive_ux is out of beta.
```

Comment:
Why don't you copy this shared example into a beta sub directory too?

Reply:
Copied an adapted to use `click_menu_link` method calls only. Please have a look.